pyregion/tests/test_region.py

```diff
@@ -1,47 +1,69 @@
 import os
-import numpy as np
 from os.path import join
 from astropy.io.fits import Header
-from ..wcs_helper import fix_lon
 from .. import open as pyregion_open
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
 
-rootdir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'examples')
+rootdir = join(os.path.dirname(os.path.abspath(__file__)), 'data')
```

Comment:
I think instead of doing this `rootdir` and path joins, it would be better to call this utility function:
http://astropy.readthedocs.io/en/latest/api/astropy.utils.data.get_pkg_data_filename.html